### PR TITLE
[1096] Add org show

### PR DESCRIPTION
**Pull request by dankmitchell**
### Context
Replica of the current c# ui 🤓 

### Changes proposed in this pull request
Add org show view

### Guidance to review
screenshot 2019-03-06 at 15 12 22

### CODE REVIEW

**Comment by misaka** [comment]
Just noticed that the `course_count` method on `Provider` isn't tested, and also the view that uses it. Seems like we're missing request/feature specs here.

File: app/controllers/providers_controller.rb [hunk 1]
```diff
@@ -4,4 +4,8 @@ class ProvidersController < ApplicationController
   def index
     @providers = Provider.all
   end
+
+  def show
+    @provider = Provider.where(institution_code: params[:id]).first
```
Comment (defong) [inline app/controllers/providers_controller.rb]: Provider.find(institution_code: param[:id])

?

**Comment by defong** [comment]
[Add org show view](https://trello.com/c/eWCanfEJ/1096-add-org-show-view)

File: app/controllers/providers_controller.rb (hunk 1 above)
Comment (dankmitchell) [reply app/controllers/providers_controller.rb]: You can't use `find` when looking up anything other than the `id` and the `json_api_client` doesnt support `find_by`

**Pull request merged**